ansible/roles/biocache-db/tasks/main.yml

```diff
@@ -2,17 +2,27 @@
 
 - name: disable swap
   shell: "sudo swapoff --all"
+  tags:
+    - biocache_db
 
 - name: copy transient files to tmp (schemas etc)
   copy: src={{item}} dest=/tmp
   with_items:
     - cassandra/cassandra-schema.txt
+  tags:
+    - biocache_db
 
 - name: restart cassandra
   service: name=cassandra state=restarted
+  tags:
+    - biocache_db
 
 - name: ensure cassandra is running
   wait_for: port=9160 delay=30
+  tags:
+    - biocache_db
 
 - name: create schema
   shell: "cassandra-cli < /tmp/cassandra-schema.txt"
+  tags:
```

Comment:
What happens when this script is run against an existing DB? I think the ansible shell task executes every time, so you would blow away an existing DB. If that's the case, then you'll need to add a ```when``` condition to this task so it only runs if the schema does not already exist.